pygmt/src/meca.py

```diff
@@ -275,23 +284,21 @@ def update_pointers(data_pointers):
             "plot_latitude": plot_latitude,
         }
 
-        # make a DataFrame copy to check convention if it contains
-        # other parameters
-        if isinstance(spec, (dict, pd.DataFrame)):
```

Comment:
Just noticed that this if-statement is unnecessary, because this code block is already nested under an `if isinstance(spec, (dict, pd.DataFrame))` statement at L143 above, and there doesn't seem to be any modifications made to the `spec` variable in between. Please double-check if this looks ok.

Reply:
Looks good to me.

Reply:
Nice catch. Looks great